monitor-iot-data-with-freeboard/README.md

```diff
@@ -0,0 +1,210 @@
+# How to create a realtime IoT dashboard for Kuzzle
+
+## Table of Content
+
+<!-- TOC -->
+
+- [How to create a realtime IoT dashboard for Kuzzle](#how-to-create-a-realtime-iot-dashboard-for-kuzzle)
+    - [Table of Content](#table-of-content)
+    - [Purpose](#purpose)
+    - [Prerequisite](#prerequisite)
+    - [Howto steps](#howto-steps)
+        - [Step 0: Getting Kuzzle Ready](#step-0-getting-kuzzle-ready)
+        - [Step 1: Enable Kuzzle Datasource Plugin](#step-1-enable-kuzzle-datasource-plugin)
+        - [Step 2: Kuzzle Datasource Configuration](#step-2-kuzzle-datasource-configuration)
+        - [Step 3: Adding datasource for the SIN_GENERATOR device](#step-3-adding-datasource-for-the-sin_generator-device)
+        - [Step 4: Adding a widget to visualize data from SIN_GENERATOR device](#step-4-adding-a-widget-to-visualize-data-from-sin_generator-device)
+        - [Step 5: Add other devices](#step-5-add-other-devices)
+    - [Conclusion](#conclusion)
+
+<!-- /TOC -->
+
+## Purpose
+
+The purpose of this howto is to learn how to create a realtime visualization dashboard for your [Kuzzle](http://www.kuzzle.io)-based by adding a simple plugin to [freeboard](http://freeboard.io), an open source dashboard.
+
+We will be using:
+
+- an instance of *Kuzzle Server* running locally
+- Kuzzle Freeboard Datasource plugin available on github: [http://github.com/kuzzleio/kuzzle-freeboard-plugin](http://github.com/kuzzleio/kuzzle-freeboard-plugin)
+- *freeboard* running locally
+
+We will be demonstrating how to use Kuzzle Freeboard Datasource plugin to monitor IoT devices. The IoT devices will be simulated by two scripts that will generate data in the way reel IoT devices would. One will generate a sinus function, while the other will generate a square function.
+
+## Prerequisite
+
+To follow this howto, you will need:
+
+- to have an instance of **Kuzzle** up and running on your computer. Refer to the *Running Kuzzle* section of this guide to install **Kuzzle**: [https://docs.kuzzle.io/](https://docs.kuzzle.io/)
+
+- Get **freeboard** from github:
+
+```console
+$ git clone https://github.com/Freeboard/freeboard.git
+```
+
+- Install **freeboard**
+
+```console
+$ cd freeboard && npm install
+```
+
+- Get Kuzzle Feeboard Datasource plugin
+
+```console
+$ git clone https://github.com/kuzzleio/kuzzle-freeboard-plugin.git
+```
+
+It is recommended to clone both repositories from the same folder so that you don't have to adapt paths from when following the steps.
```

Comment:
We recommend that you clone both repositories into the same folder so that your directory tree matches this how-to.